KEEP/services/hiawatha

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if ! grep -q '^www-data:' /etc/passwd ; then
+	busybox addgroup -S www-data
+	busybox adduser -h "$SRVDIR" -s /bin/nologin -S -D -H -G www-data www-data
+	if [ ! -d "$SRVDIR" ] ; then
+		install -D -d -m 755 "$SRVDIR"
+		chown -R www-data "$SRVDIR"
+		chgrp -R www-data "$SRVDIR"
+	fi
+fi
+
```

Comment:
pls put this here:
```[ "$1" = "--prereqs" ] && exit 0```
this prevents running the service when the --prereqs flags is passed to prepare a r/o fs